modules/ztp-cluster-provisioning.adoc

```diff
@@ -5,10 +5,12 @@
 [id="ztp-cluster-provisioning_{context}"]
 = Cluster provisioning
 
-Zero touch provisioning (ZTP) provisions clusters using a layered approach.
-The base components consist of {op-system-first}, the basic operating system
-for the cluster, and {product-title}. After these components are installed,
-the worker node can join the existing cluster. When the node has joined the existing cluster, the 5G RAN profile Operators are applied.
+Zero touch provisioning (ZTP) provisions clusters using a layered approach. The base components consist of {op-system-first}, the basic operating system
```

Comment:
I doubt there's another word we can use, but my brain really has trouble seeing provisioning provisions.

Reply:
Yes, I understand. Keep in mind that this text has already been reviewed- it's the 4 images that are new.